utils/neuroscope.py

```diff
@@ -267,6 +267,10 @@ def _plot_topk(
     # create a mask for the inclusions/exclusions
     if exclusions is not None:
         mask: Bool[Tensor, "row pos"] = get_batch_pos_mask(exclusions, dataloader, model, all_activations)
+        # get activations device
+        activations_device = activations.device
+        # put mask on same device as activations
+        mask = mask.to(activations_device)
```

Comment:
Might make more sense to move this inside `get_batch_pos_mask`. Currently that function sets the device using the model, seems like it would be better to set the device using the activations tensor

Reply:
done